Live/LiveEventWithDVR/Program.cs

```diff
@@ -79,522 +74,375 @@
 var armClient = new ArmClient(credential);
 var mediaServicesAccount = armClient.GetMediaServicesAccountResource(mediaServicesResourceId);
 
-string uniqueness = Guid.NewGuid().ToString().Substring(0, 13); // Create a GUID for uniqueness. You can make this something static if you dont want to change RTMP ingest settings in OBS constantly.  
-string liveEventName = "liveevent-" + uniqueness; // WARNING: Be careful not to leak live events using this sample!
-string assetName = "archiveAsset" + uniqueness;
-string liveOutputName = "liveOutput" + uniqueness;
-string drvStreamingLocatorName = "streamingLocator" + uniqueness;
-string archiveStreamingLocatorName = "fullLocator-" + uniqueness;
-string drvAssetFilterName = "filter-" + uniqueness;
-string streamingLocatorName = "streamingLocator" + uniqueness;
-string streamingEndpointName = "default"; // Change this to your specific streaming endpoint name if not using "default"
-bool stopEndpoint = false;
-
-// In this sample, we use Event Grid to listen to the notifications through an Azure Event Hub. 
-// If you do not provide an Event Hub config, the sample will fall back to polling the job for status. 
-// For production ready code, it is always recommended to use Event Grid instead of polling on the Job status. 
-
-EventProcessorClient? processorClient = null;
-BlobContainerClient? storageClient = null;
-MediaServicesEventProcessor? mediaEventProcessor = null;
-
-try
+var uniqueness = Guid.NewGuid().ToString()[..13]; // Create a GUID for uniqueness. You can make this something static if you don't want to change RTMP ingest settings in OBS constantly.
+var liveEventName = "liveevent-" + uniqueness;
+var assetName = "archiveAsset" + uniqueness;
+var liveOutputName = "liveOutput" + uniqueness;
+var drvStreamingLocatorName = "streamingLocator" + uniqueness;
+var archiveStreamingLocatorName = "fullLocator-" + uniqueness;
+var dvrAssetFilterName = "filter-" + uniqueness;
+var streamingLocatorName = "streamingLocator" + uniqueness;
+var streamingEndpointName = "default"; // Change this to your specific streaming endpoint name if not using "default"
+var manifestName = "output";
+
+MediaServicesMonitor? mediaServicesMonitor = null;
+
+// Optionally use Event Hub to monitor the Live Event
+if (!string.IsNullOrWhiteSpace(options.EVENT_HUB_NAME) && !options.EVENT_HUB_NAME.StartsWith("---"))
 {
-    #region CreateLiveEvent
```

Comment:
You have to preserve the #region names or the documentation will break.  Make sure that you did not remove the region names  and re-located them to the appropriate code blocks that we want displayed in the doc pages.

Reply:
Added regions back.